compiler/rustc_middle/src/ty/context.rs

```diff
@@ -1267,22 +1227,26 @@ impl<'tcx> TyCtxt<'tcx> {
         if let Some(id) = id.as_local() {
             self.hir().def_path(id)
         } else {
-            self.cstore.def_path(id)
+            self.untracked_resolutions.cstore.def_path(id)
         }
     }
 
     /// Returns whether or not the crate with CrateNum 'cnum'
     /// is marked as a private dependency
     pub fn is_private_dep(self, cnum: CrateNum) -> bool {
```

Comment:
Should this include `untracked` in the name?

Reply:
I added untracked in the name, as a warning to the reader that this value is not tracked by the query system, and may cause silent invalidations.